x/msgfee/init_test.go

```diff
@@ -0,0 +1,54 @@
+package msgfee
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/store"
+)
+
+func TestGenesis(t *testing.T) {
+	const genesis = `
+{
+	"msgfee": [
+		{
+			"msg_path": "foo/bar",
+			"fee": {"whole": 1, "fractional": 2, "ticker": "DOGE"}
+		},
+		{
+			"msg_path": "a/b",
+			"fee": {"whole": 2, "fractional": 0, "ticker": "ETH"}
+		}
+	]
+}
+	`
+	var opts weave.Options
+	if err := json.Unmarshal([]byte(genesis), &opts); err != nil {
+		t.Fatalf("cannot unmarshal genesis: %s", err)
+	}
+
+	db := store.MemStore()
+	var ini Initializer
+	if err := ini.FromGenesis(opts, db); err != nil {
```

Comment:
We need to standardize on a test library....

I was using testify/assert, and I think Alex as well.
You use stdlib test, which is quite similar, but I feel requires more typing
GoConvey is quite a break using a different paradigm than table-driven tests.

Note below that you use `%s` for the error, not `%+v`, which would show the stacktrace.
And `asset.NoError(err)` (here `require.NoError(err)` would automatically give you all this info).

But yes, this has nothing to do with the PR and is a long outstanding issue

Reply:
There is already an issue that is waiting for a hero to pick up the "fight" https://github.com/iov-one/weave/issues/249 :crossed_swords: :muscle: 

_(best to read it while listen to https://www.youtube.com/watch?v=OBwS66EBUcY)_